lld/MachO/BPSectionOrderer.cpp

```diff
@@ -60,6 +61,25 @@ getRelocHash(const Reloc &reloc,
   return getRelocHash(kind, sectionIdx.value_or(0), 0, reloc.addend);
 }
 
+static uint64_t getUnwindInfoEncodingHash(const InputSection *isec) {
```

Comment:
Can you add a quick comment explaining why adding this unwind info hash can improve uncompressed size?